person.rb

```diff
@@ -39,6 +40,62 @@ def correct_name()
   end
 end
 
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students << student
+    # Set the classroom for the student
+    student.classroom = self
+  end
+end
+
+class Student
+  attr_accessor :name, :classroom
+
+  def initialize(name)
+    @name = name
+    @classroom = nil
+  end
+
+  def classrooms=(classroom)
+    @classroom&.students&.delete(self)
+    @classroom = classroom
+    classroom.students << self
+  end
+end
+
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
Great job @boniface193 🤝

- [ ] As stated in the project requirements, kindly ensure a method for adding rentals should be present. You may use this [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) for more clarity. And, the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association.
![add-rental book](https://github.com/boniface193/oop_ruby/assets/63917725/f095b4f7-1d0e-4da2-afaa-be517c9fadce)

Reply:
please can you put more light so that i can understand